guides/source/active_job_basics.md

```diff
@@ -406,6 +466,30 @@ end
 [`around_perform`]: https://api.rubyonrails.org/classes/ActiveJob/Callbacks/ClassMethods.html#method-i-around_perform
 [`after_perform`]: https://api.rubyonrails.org/classes/ActiveJob/Callbacks/ClassMethods.html#method-i-after_perform
 
+### Callbacks with Bulk Enqueuing
```

Comment:
We have the "Callbacks" section above and the "Callbacks with Bulk Enqueuing" section here. Should they be merged into something like "Callbacks and Events" or perhaps we can keep them as two separate sections like "Callbacks" and "Events"?